frontend/.eslintrc

```diff
@@ -327,6 +327,22 @@
           }
         ]
       }
+    },
+    {
+      "files": ["src/__tests__/cypress/**"],
+      "rules": {
```

Comment:
The existing rule has this exclusion: `"excludedFiles": ["src/__mocks__/**", "src/__tests__/**"],`

Should we just remove `"src/__tests__/**"` instead of cloning the rule? I suspect we still need to exclude mocks.

Reply:
So the intention for doing this was to change the text in the error message to reflect how we could call on the environment variable in Cypress. I just couldn't find a way to do it when using it in only one usage.

Reply:
I didn't realize you had changed the message. This is fine then.